pom.xml

```diff
@@ -456,7 +446,7 @@ under the License.
                 <mockRepo>
                   <source>src/it/mrm/repository</source>
                 </mockRepo>
-                <!-- 
+                <!--
```

Comment:
why not just delete the commented element?

Reply:
The comment is in the master branch so I just left it in in case there was a reason for why someone else added it.